src/components/Transactions.tsx

```diff
@@ -31,27 +31,36 @@ const TransactionRow = observer(function ({
   const transactionsStore = React.useContext(RootContext).transactionStore;
 
   // Helper
-  const toggleSelection = function (event: React.FormEvent<HTMLInputElement>) {
-    transactionsStore.toggleSelectedTransaction(event.currentTarget.name);
+  const toggleSelection = function (
+    event: React.MouseEvent<HTMLTableRowElement>,
+  ) {
+    transactionsStore.toggleSelectedTransaction(event.currentTarget.id);
   };
 
   // Render
   return (
-    <tr className="h-8 even:bg-gray-100 hover:bg-mblue-light">
-      <td>
+    <tr
+      id={id}
+      className="h-8 even:bg-gray-100 hover:bg-mblue-light"
+      onClick={toggleSelection}
+    >
+      <td className="group relative">
         <label className="block pl-2">
           <input
             type="checkbox"
             name={id}
             className="text-mblue"
-            onChange={toggleSelection}
+            onChange={() => {}}
```

Comment:
A supprimer

Reply:
Understood, to update in pages refactor.